csharp.html.markdown

```diff
@@ -18,13 +19,15 @@ C# is an elegant and type-safe object-oriented language that enables developers
 Multi-line comments look like this
 */
 /// <summary>
-/// This is an XML documentation comment
+/// This is an XML documentation comment which can be used to generate external
+/// documentation or provide context help within an IDE
 /// </summary>
+//public void MethodOrClassOrOtherWithParsableHelp() {}
 
 // Specify namespaces application will be using
 using System;
 using System.Collections.Generic;
-using System.Data.Entity;
+using System.Data.Entity; // Add dll reference with NuGet: Install-Package EntityFramework
```

Comment:
Yes please. After you explained that I understood what was going on but until then, I didn't understand this.